Library/Homebrew/utils/json.rb

```diff
@@ -1,4 +1,4 @@
-require "vendor/okjson"
+require "json"
 
 module Utils
   module JSON
```

Comment:
Sounds good to me; that would avoid the ugly `::JSON` scoping. I'll have a PR ready tonight or tomorrow.